src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
+
```

Comment:
it would be great to use variable color, and move basic font styles to html tag so remove basic font styles in font below, (
if necessary, change the appearance of the font in a specific rule. Also use inherit in places where there is no inheritance)